docs/examples/viz_fine_tuning_gl_context.py

```diff
@@ -0,0 +1,108 @@
+"""
+=======================================================
+Fine-tuning the OpenGL state using shader callbacks
+=======================================================
+
+VTK it’s powerfully, but sometimes we need to get more control about how
+OpenGL will render the actors.  For example, enforcing that deep-test keep
+disabled during the draw call of a specific actor.  This can be useful when
+we need to fine-tuning the performance or create specific visualization
+effects in order to understand a certain data, like to enhance the
+visualization of clusters in a network.
+"""
+
+###############################################################################
+# First, let's import some functions
+import numpy as np
+
+from fury.shaders import shader_apply_effects
+
+from fury import window, actor
+import itertools
+
+
+###############################################################################
+# We just proceeds as usual: creating the actors and initializing a scene in
+# FURY
+
+centers = 1*np.array([
+    [0, 0, 0],
+    [-1, 0, 0],
+    [1, 0, 0]
+])
+centers_no_depth_test = centers - np.array([[0, -1, 0]])
+centers_additive = centers_no_depth_test - np.array([[0, -1, 0]])
+colors = np.array([
+    [1, 0, 0],
+    [0, 1, 0],
+    [0, 0, 1]
+])
+
+actors = actor.sphere(
+    centers, opacity=.5, radii=.4, colors=colors)
+actors_no_depth_test = actor.sphere(
+    centers_no_depth_test, opacity=.5, radii=.4, colors=colors)
+actor_add = actor.sphere(
+    centers_additive, opacity=1, radii=.4, colors=colors)
+
+renderer = window.Scene()
+scene = window.Scene()
+
+showm = window.ShowManager(scene,
+                           size=(900, 768), reset_camera=False,
+                           order_transparent=True)
+
+###############################################################################
+# All actors must be added  in the scene
+
+scene.add(actors)
+scene.add(actor_add)
+scene.add(actors_no_depth_test)
+###############################################################################
```

Comment:
missing the empty line above. Otherwise, the documentation will not render correctly